main.lua

```diff
@@ -9,7 +9,7 @@ return {
                 timeout = 5,
             })
         else
-            permit = ya.hide()
+            permit = ui.hide and ui.hide() or ya.hide()
```

Comment:
Here it should only be `ui.hide() or ya.hide()`, https://github.com/sxyazi/yazi/pull/2939 doesn't mention anything about the necessity of `ui.hide`